debian/control

```diff
@@ -9,6 +9,7 @@ Architecture: all
 Recommends: maratona-desktop-latam, maratona-submission
 Pre-Depends: maratona-essential
 Depends: ssh, maratona-conflitos, maratona-background, maratona-usuario-icpc, maratona-firewall, dconf-cli
+Conflicts: gnome-initial-setup
```

Comment:
Os conflitos foram removidos dos pacotes. Isso deveria ser controlado pelo task. Conflito deve acontecer apenas se for realmente essencial.